JuanGro/Balarco
# Task/igualas crud

#### What's this PR do?
Adds the whole CRUD for Igualas

#### Where should the reviewer start?
In the Igualas component

#### How should this be manually tested?
Add a new iguala
Update that Iguala
Delete Iguala
Check the cancel of form doesn't affect TWDB

#### Any background context you want to provide?
No

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
